source/includes/install-uwsgi.rst

```diff
@@ -15,7 +15,7 @@ Create  ``~/etc/services.d/uwsgi.ini`` with the following content:
   :emphasize-lines: 2
 
   [program:uwsgi]
-  command=/home/<username>/.local/bin/uwsgi --master --emperor /home/<username>/uwsgi/apps-enabled
+  command=%(ENV_HOME)s/.local/bin/uwsgi --master --emperor /home/<username>/uwsgi/apps-enabled
```

Comment:
the 2nd `<username>` can be replaced with `$(ENV_HOME)s` as well. Thus, the whole block can go :)